### PR TITLE
feat: added Tezos to universal provider

### PR DESCRIPTION
**Pull request by dianasavvatina**
## Description

Tezos is added to UniversalProvider based on other blockchains in UniversalProvider

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How has this been tested?

Used in example

## Checklist

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: providers/universal-provider/test/shared/WalletClient.ts [hunk 1]
```diff
@@ -268,6 +268,14 @@ export class WalletClient {
               );
               result = signedDirect.signature;
               break;
+            case "tezos_send":
+              //  eslint-disable-next-line no-case-declarations
+              const send_res = await this.tezosWallet.send(
```
Comment (ajinkyasraj) [inline providers/universal-provider/test/shared/WalletClient.ts]: where is this.tezoswallet defined?

Comment (ajinkyasraj) [reply providers/universal-provider/test/shared/WalletClient.ts]: we have this.cosmosWallet defined in line 29 in this file. But no tezos wallet defined.

Comment (dianasavvatina) [reply providers/universal-provider/test/shared/WalletClient.ts]: the unit tests are broken for UniversalProvider. They are hanging on init and after several hours of debugging I decided to just leave them as is. No need to make Tezos more covered than any other blockchain. Manual testing passed with example dapp

**Review by ajinkyasraj — APPROVED** [review]
LGTM

**Pull request merged**